### PR TITLE
fixed chart extending secret config

### PR DESCRIPTION
**Pull request by pedroparraortega**
## Hey, I just made a Pull Request!

Allow users to configure existing secret for env vars in order to make possible add new data to the secret
[https://github.com/backstage/backstage/issues/10223#issue-1170677611](url)

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by jhaals** [comment]
Thanks @pedroparraortega. Kindly requesting @martina-if to review and other helpful helm users.

**Comment by jhaals** [comment]
fixes #10223

**Review by jhaals — COMMENTED** [review]
The values file need formatting with prettier.

File: contrib/chart/backstage/values.yaml [hunk 1]
```diff
@@ -35,6 +35,25 @@ backend:
     limits:
       memory: 1024Mi
 
+  # `existingSecret` is a reference to an existing secret containing the sensitive information for the app-config. Make sure the secrets define the required information. EXAMPLE:
```
Comment (martina-if) [inline contrib/chart/backstage/values.yaml:38]: Should the example text for the secret here contain some dummy values or at least make a note that some of the values, if they are missing, they can make backstage fail to start? (at least IIRC that was the case for some)

**Review by martina-if — APPROVED** [review]
If you have tested both cases (with and without the new values field) I'm happy with it 👍  I just added a suggestion of something that could help future users.

**Pull request closed without merging**